x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -171,10 +171,7 @@ Privilege to index documents. Also grants access to the update mapping action.
 However, it does not enable a user to update existing documents.
 +
 --
-NOTE: When indexing documents with an external `_id` either via the index API or
-the bulk API, the request must use `op_type` as `create`. If `_id`s are
-generated automatically, the authorization happens as if the `op_type` is set to
-`create`.
+NOTE: This privilege requires the `op_type` as `create` on the Index or Bulk API.
```

Comment:
I don't feel like this explanation is really going to make sense to the typical user. The wording that is being replaced was clearer (even though out of date).

My suggestion is below, but I'd like @lcawl's input.

```suggestion
NOTE: This privilege relies on the `op_type` (_link here_) of indexing requests. When ingesting documents as a user who has the `create_doc` privilege (and no higher privilege such as `index` or `write`), the `op_type` can be set to `create` through one of the following:
1. Explicitly setting the `op_type` parameter (_link_) on the Index API or Bulk API
2. Calling the `_create` endpoint (_link_)
3. Creating a document with an auto-generated `_id` (_link_?)
```